react/react1/week3/react1week3-todos/src/components/DataFetch.js

```diff
@@ -0,0 +1,20 @@
+// data fetch
+
+import { useEffect } from 'react';
+const Api =
+  'https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw';
+
+export function DataFetch({ setFetchTodo }) {
```

Comment:
You have to export this function as a function. This is not a component. You can read my description below
```suggestion
export function dataFetch(setFetchTodo) {
```